component/main.jsonnet

```
@@ -84,7 +89,7 @@ local userParsersFile =
     [ userParsers ];
 
 local customParsersFragment =
-  if std.length(parsers) > 0 then
+  if std.length(parsers) > 0 || std.length(multiline_parsers) > 0 then
```

Alternatively, merge the regular and multiline parsers before:

```jsonnet
local all_parsers =
  local ps = (
    if std.length(parsers) > 0 then [parsers] else []
  ) + (
    if std.length(multiline_parsers) > 0 then [multiline_parsers] else []
  );
  std.join('\n', ps);
```

and here

```suggestion
  if std.length(all_parsers) > 0 then
```